modules/generator/validation/fields.go

```diff
@@ -0,0 +1,99 @@
+package validation
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/grafana/tempo/modules/generator/processor"
+	"github.com/grafana/tempo/modules/generator/registry"
+	"github.com/grafana/tempo/pkg/sharedconfig"
+	"github.com/prometheus/common/model"
+)
+
+var SupportedProcessors = []string{
```

Comment:
nit: any reasons to put it here instead of `modules/generator/processor/processor_names.go`